server/src/controllers/drone-route.controller.ts

```diff
@@ -0,0 +1,32 @@
+import { Controller, Get, Query } from '@nestjs/common';
+import { DroneRouteService } from '../services/drone-route.service';
+
+@Controller()
+export class DroneRouteController {
+  constructor(private readonly droneRouteService: DroneRouteService) {}
+
+  //add drone route info to drone route table in database
```

Comment:
Change to // add drone route info to drone route table in database

Reply:
done